Product/component/PInfo.jsx

```diff
@@ -1,23 +1,93 @@
 import React from 'react';
 import styles from './PInfo.module.css';
+import Style from './Style.jsx';
+import Option from './Option.jsx'
+import Feature from './Feature.jsx'
 
-const Info = () => (
-  <section>
-    <div className={styles.info}>
-      <div className={styles.reviews}>Reviews</div>
-      <p className={styles.category}>CATEGORY</p>
-      <p className={styles.name}>Expanded Product Name</p>
-      <p>$369</p>
-      <div className="sytle-selector">Style Selector</div>
-      <select>Size</select>
-      <select>Quantity</select>
-      <button>Checkout</button>
-      <button>favorite</button>
-      </div>
-      <div className={styles.description2}>
-        <ul>Product descripton2</ul>
+class Info extends React.Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      quantity: {},
+      skus: this.props.defaultStyle
+    }
+
+    this.addSizes = this.addSizes.bind(this)
+  }
+
+  addSizes(skus) {
+    this.setState({
+      skus: skus
+    })
+  }
+
+  render () {
+
+    return (
+      <div>
+        <div className={styles.left}>
+          <div className={styles.info}>
+            <div className={styles.reviews}>
+            ★★★★☆ <a className={styles.link}>Read all reviews</a>
+            </div>
+
+            <div className={styles.category}>
+              <p>{this.props.info.category}</p>
+            </div>
+
+            <div className={styles.name} >
+              <p className={styles.name}>{this.props.info.name}</p>
+            </div>
+
+            <div className={styles.price}>
+              <p>{'$' + this.props.info.default_price}</p>
+            </div>
+
+            <div className={styles.text}>
+              <p><strong>STYLE ></strong></p><p className={styles.second}>SELECTED STYLE</p>
+            </div>
+
+            <div className={styles.styles}>
+              {this.props.styles.map(style =>
+              <Style style={style} changeStyle={this.props.changeStyle} addSizes={this.addSizes}/>)}
+            </div>
+
+            <div className={styles.selectors}>
+              <div className={styles.select} >
+                <select>
+                  <option className={styles.option}>SELECT SIZE</option>
+                </select>
+              </div>
+              <div className={styles.quantity}>
+                <select>
+                  <option>-</option>
+                </select>
+              </div>
+            </div>
+
+            <div className={styles.checkout}>
+              <button className={styles.cart}>ADD TO BAG</button>
+              <button className={styles.favorite}>☆</button>
+            </div>
+
+          </div>
+
+          <div className={styles.description2}>
+            <ul>
+              {this.props.features.map(feature =>
+                <Feature feature={feature.feature}/>
```

Comment:
May want to add a key property here otherwise dev console will throw unique key per child warning